### PR TITLE
Issue#31. Solution for multilingual app

### LINKED ISSUE #31: Create button to select language of app and its functional
### User story
As user,
I want to change a language of application interface (Russian, Uzbek, English) 

### Tasks
- [ ] Create button with drop down menu to select language of application interface (Russian, Uzbek, English) 
- [ ] when language is changed all user interface elements and information must be in this language 
- [ ] language by default - russian

### Acceptance criteria
- [ ] language drop menu button is created and user can full full translated version of app

### PR DESCRIPTION
**Pull request by Ivan-Andr**
Closes https://github.com/Ivan-Andr/psychological-test/issues/31
Add solution for translation page to uzbek and english language. Translation.js file will be updated in the process of development

### CODE REVIEW

File: src/ui/lang.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"use strict";
+import { langArr } from "./translation.js";
+const langSelect = document.querySelector(".change-lang");
+const languages = ["ru", "en", "uz"];
+langSelect.addEventListener("change", changeURLLanguage);
+export function changeURLLanguage() {
```
Comment (levnikan) [inline src/ui/lang.js]: since this function is only used internally, you don't need to export it

File: src/ui/lang.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  hash = hash.substring(1);
+  console.log(hash);
+  if (!languages.includes(hash)) {
+    location.href = window.location.pathname + "#ru";
+    location.reload();
+  }
+  langSelect.value = hash;
+  // document.querySelector(".first-name").innerHTML = langArr["first-name"][hash];
+  for (let key in langArr) {
+    let element = document.querySelector(`.${key}`);
```
Comment (levnikan) [inline src/ui/lang.js]: why do you use classes and not ids, for example? Otherwise looks like class is not used for its main purpose.

File: src/ui/translation.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    ru: "Начать тест",
+    uz: "Testni boshlang",
+    en: "Start test",
+  },
+  "btn--finish": {
+    ru: "Завершить тест",
+    uz: "Testni tugatish",
+    en: "Finish test",
+  },
+  "test-timer": {
```
Comment (levnikan) [inline src/ui/translation.js:82]: I think you forgot about seconds translation

File: src/ui/lang.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+"use strict";
+import { langArr } from "./translation.js";
+const langSelect = document.querySelector(".change-lang");
+const languages = ["ru", "en", "uz"];
+langSelect.addEventListener("change", changeURLLanguage);
+export function changeURLLanguage() {
+  let lang = langSelect.value;
+  location.href = window.location.pathname + "#" + lang;
+  location.reload();
```
Comment (levnikan) [inline src/ui/lang.js]: I think it's better to replace `location.reload()` with `changeLanguage()` here, what do you think? This way we don't need to reload the page. Generally, manual reload of the page should be avoided (might lead to weird unexpected bugs), unless there are no other options.

**Review by levnikan — COMMENTED** [review]
Looks good and relatively easy! Please check my comments.

File: src/ui/lang.js (hunk 4 above)
Comment (Ivan-Andr) [reply src/ui/lang.js]: I tried to fix bugs. Please, test it one more time

File: style.css [hunk 5]
```diff
@@ -77,17 +77,17 @@ td:hover {
   cursor: pointer;
   font-family: inherit;
 }
-.btn--start:hover,
-.btn--next:hover {
+#btn--start:hover,
```
Comment (levnikan) [inline style.css]: why did you change classes here, let's leave as it was before and use id's for the purpose of translation

**Review by levnikan — DISMISSED** [review]

File: style.css (hunk 5 above)
Comment (Ivan-Andr) [reply style.css]: Ok I'll fix it

**Review by levnikan — APPROVED** [review]
